_posts/en/2018-08-30-dashboard-announcement.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Dashboard'
+permalink: /en/dashboard-announcement/
+name: 2018-08-30-dashboard-announcement
+type: posts
+layout: post
+lang: en
+version: 1
+---
+
+{:.post-meta}
+*by [Marcin Jachymiak](https://github.com/marcinja)<br>Intern at Bitcoin Optech*
+
+This summer I was an intern for [Bitcoin Optech](https://bitcoinops.org), working on a Bitcoin metrics dashboard. In this post I'll be describing the purpose of the dashboard and how it was implemented. 
+
+The goal of the Optech dashboard is to show a variety of metrics of how effectively blockspace is being used. Important metrics are those that show activity like: 
+ - [batching](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Payment_batching): combining the outputs of what would otherwise be many different transactions into a single transaction
+ - [consolidations](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Consolidation): combining many UTXOs in a single UTXO during low fee period, to avoid paying a higher fee in the future
+ - dust creation: creation of UTXOs that cost more to spend than they have value, at different fee-rates
+ - SegWit adoption: creation of and spending from SegWit outputs
+ 
+
+The dashboard is live and is showing these stats and more at [dashboard.bitcoinops.org](https://dashboard.bitcoinops.org).
+
+The dataset used for the dashboard is also saved as a directory of JSON files, updated nightly. You can download the zipped dataset from our [S3 bucket](http://dashboard.dataset.s3.us-east-2.amazonaws.com/backups/bitcoinops-dataset.tar.gz). If you just want the JSON file for a specific block, you can get it at: `http://dashboard.dataset.s3.us-east-2.amazonaws.com/blocks/BLOCK_NUMBER.json` If you'd rather recreate the data using your own full node (it might take a few days), you can use the code with instructions at [github.com/bitcoinops/btc-dashboard](https://github.com/bitcoinops/btc-dashboard).
```

Comment:
missing . after the JSON URL. I'd also change recreate -> regenerate (sorry - I know I suggested recreate earlier)